fdbclient/include/fdbclient/GetEncryptCipherKeys.actor.h

```diff
@@ -292,6 +292,11 @@ struct TextAndHeaderCipherKeys {
 	Reference<BlobCipherKey> cipherHeaderKey;
 };
 
+struct TextAndHeaderCipherKeysOpt {
```

Comment:
nit: we should rather update the exiting `struct` and update the call sites; in future we would need in anyways.

Reply:
I mean regardless of `configurable encryption` the `headerCipher` key as optional is valid

Reply:
Yeah the only reason i avoid this for now was because it was going to cause alot of stuff to need changing to check for presence of header. Maybe once we finish configurable encryption and when we remove the old code we can make the new default